mp/src/game/client/momentum/ui/SettingsPanel/RocketJumpSettingsPage.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "SettingsPage.h"
+
+class C_RJSettingsPanel;
```

Comment:
What is this line for?

Reply:
I looked at how the other settings panels were made and I didn't realise that this wasn't needed as the other panels like the Appearance tab use it for the model preview.